tacit-dark-theme.css.map

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
@akshayvinchurkar1 I believe this map was committed by mistake (You might be working on dark theme, am I right)?
If it was, I recommend you to update `.gitignore` file to ignore any `*.css.map` file.